vulcan/__init__.py

```diff
@@ -67,6 +68,7 @@ class Vulcan:
     lockfile: Path
     dependencies: Optional[List[str]]
     configured_dependencies: VersionSpecs
+    dev_dependencies: Optional[List[Requirement]]
```

Comment:
Why is `dev_dependencies` a `List[Requirement]` but `dependencies` is `List[str]`?

Reply:
Ah this PR was replaced already, vulcan2 already has support for dev dependencies. It's done differently there :D